tools/mtouch/xlf/Errors.cs.xlf

```diff
@@ -3002,6 +3002,30 @@
         <note>
 		</note>
       </trans-unit>
+      <trans-unit id="MX5312">
```

Comment:
OT: Does anyone actually translate those? I didn't even know the translation files are there.

Reply:
> Does anyone actually translate those?

Not yet, but soon.

> I didn't even know the translation files are there.

It's very new (just a few weeks ago), which is why the translations aren't there yet. Step 1 is to make it possible to translate, step 2 is to actually translate :smile:

Reply:
People have been working on step 2 for a few weeks so hopefully there will be something soon.